### ISSUE TITLE
Feature Request: Download URL including a token

### ISSUE THREAD

**Issue by chmac** [spiral-project/ihatemoney#1078]
Here's the scenario. We have a group sharing expenses for a month. As it's super easy to delete expenses (see also #1077), I'd like to download a JSON copy of our bills each day. I thought I could just grab the JSON download URL pop it into cron and wget and be done. But that doesn't work, instead I get an HTML page. It would be great if there was a token included in the JSON download which would allow me to immediately download the data without being logged in via the web UI.

**Comment by Glandos** [comment]
Hi!
For such automation, you can use the REST API https://ihatemoney.readthedocs.io/en/latest/api.html#authentication It includes documentation for basic auth as well as token based one.

Does it fit your need?

**Comment by chmac** [comment]
@Glandos Yeah, that makes sense. I didn't think about using the API, but I think that would work for our use case. Thanks a lot.
